NSS-Day-Cohort-42/NSS-Day-Cohort-42.github.io
# Cj new cohort data

## Description of Proposed Changes

- New cohort.json file for cohort 42

## Steps to Test
- `git fetch --all`
- `git checkout cj-new-cohort-data`
- run `serve` from the top level of the Class-Website directory
- verify that the names have changed to reflect the current students


## Impacted Areas in Application

List general components of the application that this PR will affect:

* cohort.json
